game_manager/backend/models.py

```diff
@@ -0,0 +1,92 @@
+from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=200)
+
+    def __str__(self):
```

Comment:
You may also want to define `__repr__` because that's used by things like printing a dict iirc.

Reply:
Good idea, but leaving out for now since this isn't used yet, and __str__ is only for debugging. Much more important coming up in subsequent changes are the `serializers`